wireapp/wire-android
# feat: Move conversation to first unread message (AR-2998)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2998" title="AR-2998" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-2998</a>  When opening a conversation set focus to the oldest unread message
  </td></table>
  <br />
 






# What's new in this PR?

When opening the conversation messages list scroll to the first (the oldest) unread message

### Dependencies (Optional)

Needs releases with:

After merging kalium PR https://github.com/wireapp/kalium/pull/1714

### Testing

#### How to Test

- receive a lot of messages in some conversation
- open that conversation
Expected result: 
messages list will be scrolled to the oldest unread message. 

- receive more then 50 messages in some conversation 
(`50` is a defaulted page size in messages pager)
- open that conversation
Expected result: 
messages list will be scrolled to the oldest unread message. 

### Attachments (Optional)


https://github.com/wireapp/wire-android-reloaded/assets/6539347/873cfdf7-638e-4780-a05d-73a5700d92fe
